docs/content/docs/testing/telemetry.md

```diff
@@ -13,23 +13,16 @@ your other telemetry, and route the event data in whatever way you need.
 ## Collect Fluid Framework logs with a custom `ITelemetryBaseLogger`
 
 The `ITelemetryBaseLogger` is an interface within the `@fluidframework/common-definitions` package. This interface can
-be implemented and passed into the client's `createContainer()` and `getContainer()` methods via the config parameter.
+be implemented and passed into the client's constructor via the props parameter.
 
-All Fluid service clients (for example, `AzureContainerConfig` and `TinyliciousContainerConfig`) allow passing a `logger?:
-ITelemetryBaseLogger` into the service client configuration. Both `createContainer()` and `getContainer()` methods will
-create an instance of a `Loader` class object, where the logger defined in the service client configuration is passed in
-as an optional parameter, `ILoaderProps.logger`, in the `Loader` constructor argument.
+All Fluid service clients (for example, `AzureClient` and `TinyliciousClient`) allow passing a `logger?: ITelemetryBaseLogger`
+into the service client props. Both `createContainer()` and `getContainer()` methods will
+create an instance of a `Loader` class object, where the logger defined in the service client props is passed in
+as an optional parameter, `ILoaderProps.logger`, to the `Loader` constructor.
```

Comment:
Are the details of what objects are created and how the logger is passed through relevant to consumers?

Reply:
from an implementation standpoint, no. This more or less just serves as providing slightly more context to the developers. But I agree this isn't diving deep enough to provide any meaningful background, and even more so now that logger is extracted away from `createContainer` and `getContainer`

Reply:
I will remove the details